server/util/ServerCommand.java

```diff
@@ -222,5 +230,36 @@ public ExportData asExportData() {
             return this;
         }
     }
+
+    @Command(name = "schema")
+    class GetSchema implements ServerCommand {
```

Comment:
Should this maybe be `PrintSchema`? And the command could be `print schema`.

Reply:
It used to be called just `schema`

Reply:
But the command for server start is called `grakn server`, right?